hawk/app/models/cib.rb

```diff
@@ -80,6 +81,119 @@ def sim?
     id != 'live'
   end
 
+  def status(minimal = false)
+    {
+      meta: meta.to_h,
+      errors: errors,
+      booth: booth
+    }.tap do |result|
+      if minimal
+        result[:resources] = {}
+
+        result[:resource_states] = {
+          pending: 0,
+          started: 0,
+          failed: 0,
+          master: 0,
+          slave: 0,
+          stopped: 0
+        }
+
+        result[:nodes] = {}
+
+        result[:node_states] = {
+          pending: 0,
+          online: 0,
+          standby: 0,
+          offline: 0,
+          unclean: 0
+        }
+
+        result[:tickets] = []
+
+        result[:ticket_states] = {
+          granted: 0,
+          revoked: 0
+        }
+
+        current_resources.each do |key, values|
+          result[:resources][key] ||= {}
+
+          values[:instances].each do |name, attrs|
```

Comment:
Unused block argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used.